### DISCUSSION TITLE
Upload in relation with subfields

### DISCUSSION THREAD

**Discussion by miquelangeld** [Laravel-Backpack/community-forum#474] [General]
I created [this](https://github.com/Laravel-Backpack/community-forum/issues/406) issue which was closed  by https://github.com/Laravel-Backpack/CRUD/pull/4921 but my problem persist using upload and multiple uploads using it in a relationship with subfields. Was this finally merged?

Thanks!

**Comment by pxpm** [comment]
Hello @miquelangeld thanks for the suggestion.

It was not simple to fix, it was not just a "simple bug". 
We tackled it, **in v6 uploads are completely revamped**, and work inside relationship fields managed with the repeatable interface. (v6 alpha should be released very soon)

I am sorry it took a little bit more time, but like I told you, it required massive changes and tinkering on how we do stuff to properly make uploads work in an intuitive way. 

https://github.com/Laravel-Backpack/CRUD/pull/4988 is the PR that replaced https://github.com/Laravel-Backpack/CRUD/pull/4921 and it's already merged on v6 👍 

Let me know if I can help you with something else.
